src/extensions/extra/PDFView/styles/viewer-overrides.css

```diff
@@ -2,6 +2,16 @@
  * Various overrides of the built viewer.css file. Keep this as a CSS file vs. LESS
  */
 
+
+#pdf-meta *::selection  {
+    background: #abdffa !important;
```

Comment:
In testing this, it seems that the rule takes effect without needing the `!important` declaration. Can you confirm on your end and remove `!important` if that is the case?

Reply:
It seems to be working fine without the !important declaration.